src/probnum/problems/zoo/filtsmooth/_filtsmooth_problems.py

```diff
@@ -0,0 +1,470 @@
+from typing import Optional, Tuple, Union
+
+import numpy as np
+
+from probnum import diffeq, filtsmooth, problems, randvars, statespace
+from probnum.type import FloatArgType
+
+__all__ = [
+    "benes_daum",
+    "car_tracking",
+    "logistic_ode",
+    "ornstein_uhlenbeck",
+    "pendulum",
+]
+
+
+def car_tracking(
+    measurement_variance: FloatArgType = 0.5,
+    process_diffusion: FloatArgType = 1.0,
+    times: Optional[np.ndarray] = None,
+    initrv: Optional[randvars.RandomVariable] = None,
+):
+    r"""Filtering/smoothing setup for a simple car-tracking scenario.
+
+    A linear, time-invariant Gaussian state space model for car-tracking, based on
+    Example 3.6 in Särkkä, 2013. [1]_
+    Let :math:`X = (\dot{x}_1, \dot{x}_2, \ddot{x}_1, \ddot{x}_2)`. Then the state
+    space model has the following discretized formulation
+
+    .. math::
+
+        X(t_{n}) &=
+        \begin{pmatrix}
+          1 & 0 & \Delta t& 0 \\
+          0 & 1 & 0 & \Delta t \\
+          0 & 0 & 1 & 0 \\
+          0 & 0 & 0 & 1
+        \end{pmatrix} X(t_{n-1})
+        +
+        q(t_n) \\
+        y_{n} &=
+        \begin{pmatrix}
+          1 & 0 & 0 & 0 \\
+          0 & 1 & 0 & 0 \\
+        \end{pmatrix} X(t_{n})
+        + r_n
+
+    where :math:`q(t_n) \sim \mathcal{N}(0, Q)` and :math:`r_n \sim \mathcal{N}(0, R)`
+    for process noise covariance matrix :math:`Q` and measurement noise covariance
+    matrix :math:`R`.
+
+    Parameters
+    ----------
+    measurement_variance
+        Marginal measurement variance.
+    process_diffusion
+        Diffusion constant for the dynamics.
+    times
+        Time grid for the filtering/smoothing problem.
+    initrv
+        Initial random variable.
+
+    Returns
+    -------
+    regression_problem
+        `RegressionProblem` object with time points and noisy observations.
+    statespace_components
+        Dictionary containing
+        - dynamics model
+        - measurement model
+        - initial random variable
+
+    References
+    ----------
+    .. [1] Särkkä, Simo. Bayesian Filtering and Smoothing. Cambridge University Press,
+        2013.
+
+    """
+
+    dynamics_model = statespace.IBM(ordint=1, spatialdim=2)
```

Comment:
I would suggest using the equivalent discretisation here, because this is what a user might expect here.